region/brinstar/green/Green Brinstar Beetom Room.json

```diff
@@ -99,11 +76,26 @@
     {
       "from": 2,
       "to": [
-        {"id": 1}
+        {"id": 1},
+        {"id": 2}
       ]
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway",
+      "notable": false,
+      "requires": [
+        {"obstaclesCleared": ["A"]}
```

Comment:
Taking damage is a pretty reasonable way to use the runway with beetoms alive.

Reply:
Yeah, I hesitated to add it, because we have to think about all the ways that a runway can be used (e.g. not just for gaining a shinecharge, but also for door stuck setups, bomb boosts, etc.). In this case I think it's fine though; the various ways to use the runway will probably have a similar number of Beetom hits; using it to start a short-charge will probably be the worst case so we could just look at that. In the future, if we need to we could refine the `leaveWithRunway` to allow expressing that only certain ways of using the runway are allowed.

Reply:
Added now.